### PR TITLE
Upgrade from Chromium 80.0.3987.66 to Chromium 80.0.3987.78.

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes brave/brave-browser#7975 
Related PR: brave/brave-browser#7979

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: patches/chrome-browser-updates-announcement_notification-BUILD.gn.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/updates/announcement_notification/BUILD.gn b/chrome/browser/updates/announcement_notification/BUILD.gn
+index fe6f065a756ddfc0bd651ec4387948a368d09867..3193cdf414407b508ae6991ad4159fb3d141d686 100644
+--- a/chrome/browser/updates/announcement_notification/BUILD.gn
++++ b/chrome/browser/updates/announcement_notification/BUILD.gn
+@@ -20,6 +20,7 @@ source_set("announcement_notification") {
+ 
+   deps = [
+     "//base",
++    "//chrome/app:generated_resources",
```
Comment (simonhong) [inline patches/chrome-browser-updates-announcement_notification-BUILD.gn.patch]: Yes, this looks like upstream bug w/o this for android build.
I think their buildbot didn't see this error by luck? :)

**Comment by simonhong** [comment]
CI only has `network-audit` failure - maybe it's `Crowd Deny`.

**Comment by mkarolin** [comment]
Yes, the CI failure in `network-audit` is consistent with `Crowd Deny` (issue brave/brave-browser#7977).

**Comment by simonhong** [comment]
FYI, filed an issue for above deps issue - https://bugs.chromium.org/p/chromium/issues/detail?id=1048346.

**Comment by simonhong** [comment]
FYI above deps issue was fixed in upstream - https://chromium-review.googlesource.com/c/chromium/src/+/2036915

**Pull request merged**